src/content/ko/fundamentals/performance/prpl-pattern/index.md

```diff
@@ -0,0 +1,197 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2016-09-28 #}
+{# wf_published_on: 2016-09-28 #}
+
+# The PRPL Pattern {: .page-title }
+
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+개요 : PRPL 은 우리가 느끼기에 굉장한 잠재력을 지닌 새로운 패턴입니다. 이번 단계에서는 이 패턴에 대한 아이디어를 토대로 반복적인 실험을 해보고 매력적인 장점을 발견할 수 있도록 더 많은 정보를 얻으려고 합니다.
```

Comment:
@petele, please correct me if I'm wrong. I believe that the word 'Dogfood' needs to remain untranslated because  it is handled by a script.